docs/guides/key-manager/give-permissions.md

```diff
@@ -128,43 +128,77 @@ The private key can be obtained depending on how you created your Universal Prof
 - UP created with the _lsp-factory.js_: this is the private key given in the `controllerAddresses` field in the method [`lspFactory.UniversalProfile.deploy(...)`](../../tools/lsp-factoryjs/classes/universal-profile#deploy)
 - UP created via the Browser extension: click on the _Settings_ icon (top right) > and _Export Private Key_
 
+<Tabs>
+  
+  <TabItem value="web3js" label="web3.js">
 
 ```javascript title="Load account from a private key"
+import UniversalProfile from '@lukso/lsp-smart-contracts/artifacts/UniversalProfile.json';
```

Comment:
these imports should not be included in this section, because this step 3.1 only deals with creating a wallet from the private key. Please remove these imports.

Like it was before.

<img width="1027" alt="image" src="https://user-images.githubusercontent.com/31145285/232020284-539a2f79-8516-45cd-98db-b4db05dc09ba.png">